packages/documentation-site/patternfly-docs/content/demos/Compass/CompassIntegrationsDemo.tsx

```diff
@@ -0,0 +1,750 @@
+                  </DescriptionListDescription>
+                </DescriptionListGroup>
+              </DescriptionList>
+            </CardBody>
+          </Card>
+        ))}
+      </Gallery>
+    </>
+  );
+
+  const rowActions = [
+    {
+      title: "Some action",
+      onClick: () => console.log("clicked on Some action"), // eslint-disable-line no-console
+    },
+    {
+      title: <div>Another action</div>,
+      onClick: () => console.log("clicked on Another action"), // eslint-disable-line no-console
+    },
+    {
+      isSeparator: true,
+    },
+    {
+      title: "Third action",
+      onClick: () => console.log("clicked on Third action"), // eslint-disable-line no-console
+    },
+  ];
+
+  const rows = integrations.map(
+    ({ name, type, status, statusText }, index) => ({
+      id: name,
+      row: [
+        {
+          id: "select",
+          row: name,
+          cell: undefined,
+          props: {
+            select: {
+              index,
+              rowIndex: index,
+              onSelect: () => { },
+              isSelected: false,
+            },
+          },
+        },
+        name,
+        type,
+        {
+          cell: (
+            <Label
+              status={
+                status as
+                | "success"
+                | "danger"
+                | "warning"
+                | "info"
+                | "custom"
+                | undefined
+              }
+              isCompact
+              variant="outline"
+            >
+              {statusText}
+            </Label>
+          ),
+        },
+        {
+          cell: <ActionsColumn items={rowActions} />,
+          props: { isActionCell: true },
+        },
+      ],
+    })
+  );
+
+  const columns = [
+    {
+      cell: undefined,
+      props: {
+        select: {
+          onSelect: () => { },
+          isSelected: false,
+        },
+      },
+    },
+    "Name",
+    "Type",
+    "Status",
+  ];
+
+  const tableViewContent = (
+    <>
+      <DataViewToolbar
+        clearAllFilters={() => { }}
+        filters={
+          <DataViewTextFilter
+            filterId="name"
+            title="Name"
+            placeholder="Filter by name"
+          />
+        }
+        actions={
+          <ToggleGroup>
+            <ToggleGroupItem
+              icon={<ThIcon />}
+              aria-label="grid icon button"
+              isSelected={activeDisplay === "card"}
+              onChange={() => setActiveDisplay("card")}
+            ></ToggleGroupItem>
+            <ToggleGroupItem
+              icon={<ListIcon />}
+              aria-label="list icon button"
+              isSelected={activeDisplay === "table"}
+              onChange={() => setActiveDisplay("table")}
+            ></ToggleGroupItem>
+          </ToggleGroup>
+        }
+        pagination={<Pagination page={1} perPage={10} isCompact />}
+      />
+      <DataViewTable
+        aria-label="Integrations table"
```

Comment:
```suggestion
        aria-label="Integrations"
```
Similar to above comment